README.md

```diff
@@ -83,6 +83,14 @@ The following is required in order to let node use the Bluetooth adapter.
 $ sudo setcap cap_net_raw+eip $(eval readlink -f `which node`)
 ```
 
+### Install Bluetooth and BT Low Energy support libraries (Raspberry Pi Only)
```

Comment:
I would change this to "Linux Only", as it will also apply to other Linux installations.